extensions/amp-video/1.0/video-wrapper.js

```diff
@@ -0,0 +1,232 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as Preact from '../../../src/preact';
+import {ContainWrapper} from '../../../src/preact/component';
+import {Deferred} from '../../../src/utils/promise';
+import {MIN_VISIBILITY_RATIO_FOR_AUTOPLAY} from '../../../src/video-interface';
+import {cssText as autoplayCss} from '../../../build/video-autoplay.css';
+import {fillContentOverlay} from './video-wrapper.css';
+import {
+  parseFavicon,
+  parseOgImage,
+  parseSchemaImage,
+  setMediaSession,
+} from '../../../src/mediasession-helper';
+import {useEffect, useMemo, useRef, useState} from '../../../src/preact';
+import {useMountEffect, useResourcesNotify} from '../../../src/preact/utils';
+
+/**
+ * @param {{getMetadata: (function():!Object|undefined)}} player
+ * @param {!VideoWrapperProps} props
+ * @return {!Object};
+ */
+function getMetadata(player, props) {
+  const metadata = player.getMetadata
+    ? player.getMetadata()
+    : {
+        title: props.title || '',
```

Comment:
Do these need to be in quotes? Any risk of these being obfuscated in an unexpected way?